cmd/ocsp-updater/main.go

```diff
@@ -51,28 +102,70 @@ func setupClients(c cmd.Config, stats statsd.Statter) (rpc.CertificateAuthorityC
 	return cac, closeChan
 }
 
-func (updater *OCSPUpdater) processResponse(tx *gorp.Transaction, serial string) error {
-	certObj, err := tx.Get(core.Certificate{}, serial)
-	if err != nil {
-		return err
-	}
-	statusObj, err := tx.Get(core.CertificateStatus{}, serial)
-	if err != nil {
-		return err
+var errNoStaleResponses = errors.New("No stale responses to update")
+
+func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Time, batchSize int) ([]core.Certificate, error) {
+	var certs []core.Certificate
+	_, err := updater.dbMap.Select(
+		&certs,
+		`SELECT cert.*
+		 FROM certificateStatus AS cs
+		 JOIN certificates AS cert
```

Comment:
This is the already-existing query, which runs in a slowish loop. You may be thinking of the query, below, which runs very frequently. See discussion down there about how to speed it up. I think tweaking the existing query for stale responses should be out of scope for this branch since the frequency doesn't change.

You might also be thinking of #824, where I suggest setting an `expired` field on certificateStatus. Also out of scope, I think.